sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -366,6 +366,44 @@
     "sample_record"
```

Comment:
<p>Please confirm with our doc writer the wording here makes sense.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>